### PR TITLE
Gradle remote build cache

### PR DESCRIPTION
**Pull request by sebersole**
Enable remote publishing of Gradle build cache from CI jobs

### CODE REVIEW

File: gradle/gradle-enterprise.gradle [hunk 1]
```diff
@@ -36,6 +36,14 @@ static java.util.Optional<String> getSetting(String name) {
     return java.util.Optional.ofNullable(sysProp);
 }
 
+static boolean isEnabled(String setting) {
+    if ( System.getenv().hasProperty( setting ) ) {
+        return true
```
Comment (Sanne) [inline gradle/gradle-enterprise.gradle:41]: isn't this going to return "true" even when setting `-DPOPULATE_REMOTE=false` ?

Comment (sebersole) [reply gradle/gradle-enterprise.gradle:41]: Its not meant to be a true/false flag

Comment (sebersole) [reply gradle/gradle-enterprise.gradle:41]: I just had to pass *some* value

**Comment by beikov** [comment]
Merged via d7c1886aaa150e4c9f740ebe907b9f4ce7138a28

**Pull request closed without merging**